### DISCUSSION TITLE
Cannot connect to S3 HTTPS endpoint

### DISCUSSION THREAD

**Discussion by ssgituser** [iterate-ch/cyberduck#15789] [Q&A]
OS: Windows 10
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/14838137/cyberduck.log)

I have an issue connecting to an S3 bucket via HTTPS (non AWS). When trying to connect I get the error message:

“Cannot read bucket location.

Connect to bucketname.hostname.subdomain.tld:443 [bucketname.hostname.subdomain.tld/10.0.0.1] failed. Connection refused: connect.
“

I have no idea where the 10.0.0.1 is coming from in the connection string.

This exact connection works on other computers with the same settings and versions (latest downloaded as of 4/1/2024). I have also attached the log file for your review. 

Thank you in advance for your time in looking into this.

**Comment by dkocher** [comment]
The system you are connecting to might not support virtual host style requests (although the bucket name can be resolved being part of the hostname which would indicate a configuration error in the DNS). Try if you can connect using the _S3 (Deprecated path style requests) profile_ connection profile [^1] you can find in _Preferences → Profiles_.

[^1]: https://docs.cyberduck.io/protocols/s3/#disable-use-of-virtual-host-style-requests

**Comment by ssgituser** [comment]
@dkocher Thank you for the tip, but why does this same connection work from other desktops where I have not disabled use of Virtual Host Style Requests?

**Comment by Solkre** [comment]
I'm also having trouble connecting to a spectralogic blackpearl with the downloaded profile.  s3.bucket.virtualhost.disable=true is already set in that profile.

**Reply by dkocher** [reply]
Did you try the _Spectra S3_ connection profile available from _Preferences → Profiles_?

**Comment by Solkre** [comment]
Yes we were.  We figured out we had to go back to 8.6 for it to work.  Now
we have a problem where it can't make folders or put files in the buckets
that require SHA-256 CRC

On Fri, Apr 12, 2024 at 4:30 AM David Kocher ***@***.***>
wrote:

> Did you try the *Spectra S3* connection profile available from *Preferences
> → Profiles*?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iterate-ch/cyberduck/discussions/15789#discussioncomment-9093103>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2PPP6ESRUX25JJEITI7EU3Y46LRTAVCNFSM6AAAAABFTQPRHWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAOJTGEYDG>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Sincerely,

Matt O'Dell
***@***.***

**Reply by dkocher** [reply]
Please reach out to _Spectralogic_ to determine what interoperability issue exists with the profile and latest release of Cyberduck.

**Comment by Solkre** [comment]
We did, I have an open ticket.  They're the ones who said to go back to
8.6  Then we ran into the hashing issue.  I hope we can get it working, we
want an alternative to Eon.

On Fri, Apr 12, 2024 at 8:43 AM David Kocher ***@***.***>
wrote:

> Please reach out to *Spectralogic* to determine what interoperability
> issue exists with the profile and latest release of Cyberduck.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iterate-ch/cyberduck/discussions/15789#discussioncomment-9095711>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2PPP6D27LLDWHPWIXYV2YDY47JIHAVCNFSM6AAAAABFTQPRHWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAOJVG4YTC>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Sincerely,

Matt O'Dell
***@***.***
